### ISSUE TITLE
grid search creates overlapping logs in tensorboard

### ISSUE THREAD

**Issue by vincentschen** [HazyResearch/metal#120]
![image](https://user-images.githubusercontent.com/5819303/50802669-354dcc80-129c-11e9-9fb6-3d562e89924c.png)

```
 ➜  ls slice-r2-advanced2-10x10
events.out.tfevents.1546851383.raiders2.stanford.edu.model_search_0   events.out.tfevents.1546866958.raiders2.stanford.edu.model_search_14  model_search_16_05_51_08.json
events.out.tfevents.1546852271.raiders2.stanford.edu.model_search_0   events.out.tfevents.1546868011.raiders2.stanford.edu.model_search_15  model_search_17_06_08_30.json
events.out.tfevents.1546853328.raiders2.stanford.edu.model_search_1   events.out.tfevents.1546869068.raiders2.stanford.edu.model_search_16  model_search_18_06_26_07.json
events.out.tfevents.1546854382.raiders2.stanford.edu.model_search_2   events.out.tfevents.1546870110.raiders2.stanford.edu.model_search_17  model_search_19_06_43_20.json
events.out.tfevents.1546855439.raiders2.stanford.edu.model_search_3   events.out.tfevents.1546871167.raiders2.stanford.edu.model_search_18  model_search_2_01_46_22.json
events.out.tfevents.1546856484.raiders2.stanford.edu.model_search_4   events.out.tfevents.1546872200.raiders2.stanford.edu.model_search_19  model_search_3_02_03_59.json
events.out.tfevents.1546857524.raiders2.stanford.edu.model_search_5   model_search_0_00_56_23.json                                          model_search_4_02_21_24.json
events.out.tfevents.1546858578.raiders2.stanford.edu.model_search_6   model_search_0_01_11_11.json                                          model_search_5_02_38_44.json
events.out.tfevents.1546859641.raiders2.stanford.edu.model_search_7   model_search_10_04_05_45.json                                         model_search_6_02_56_18.json
events.out.tfevents.1546860672.raiders2.stanford.edu.model_search_8   model_search_1_01_28_48.json                                          model_search_7_03_14_01.json
events.out.tfevents.1546861712.raiders2.stanford.edu.model_search_9   model_search_11_04_23_32.json                                         model_search_8_03_31_12.json
events.out.tfevents.1546862745.raiders2.stanford.edu.model_search_10  model_search_12_04_40_59.json                                         model_search_9_03_48_32.json
events.out.tfevents.1546863812.raiders2.stanford.edu.model_search_11  model_search_13_04_58_25.json                                         tuner_report.json
events.out.tfevents.1546864859.raiders2.stanford.edu.model_search_12  model_search_14_05_15_57.json
events.out.tfevents.1546865905.raiders2.stanford.edu.model_search_13  model_search_15_05_33_31.json
```

**Comment by jdunnmon** [comment]
This is fixed in `slicing` -- can merge to master when we update

**Comment by bhancock8** [comment]
Fixed in #134.
